<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Circle</class>
 <widget class="QDialog" name="Circle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日数据模拟</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>请选择日期</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="ruleGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>请选择规则</string>
        </property>
        <widget class="QComboBox" name="listWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>120</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableWidget" name="tableWidget"/>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <widget class="QPushButton" name="btnAreaData">
               <property name="geometry">
                <rect>
                 <x>450</x>
                 <y>10</y>
                 <width>80</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="text">
                <string>导出面积数据</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_10">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>15</y>
                 <width>54</width>
                 <height>12</height>
                </rect>
               </property>
               <property name="text">
                <string>起止日期:</string>
               </property>
              </widget>
              <widget class="QDateEdit" name="inputEndDate">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>10</y>
                 <width>91</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="displayFormat">
                <string>yyyy-MM-dd</string>
               </property>
              </widget>
              <widget class="QPushButton" name="btnImport">
               <property name="geometry">
                <rect>
                 <x>381</x>
                 <y>10</y>
                 <width>60</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>导入</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_11">
               <property name="geometry">
                <rect>
                 <x>165</x>
                 <y>10</y>
                 <width>51</width>
                 <height>12</height>
                </rect>
               </property>
               <property name="text">
                <string>至</string>
               </property>
              </widget>
              <widget class="QPushButton" name="btnCaculate">
               <property name="geometry">
                <rect>
                 <x>309</x>
                 <y>10</y>
                 <width>60</width>
                 <height>26</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>重新计算</string>
               </property>
              </widget>
              <widget class="QDateEdit" name="inputStartDate">
               <property name="geometry">
                <rect>
                 <x>67</x>
                 <y>10</y>
                 <width>91</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="displayFormat">
                <string>yyyy-MM-dd</string>
               </property>
              </widget>
              <widget class="QLabel" name="txtN">
               <property name="geometry">
                <rect>
                 <x>280</x>
                 <y>15</y>
                 <width>31</width>
                 <height>12</height>
                </rect>
               </property>
               <property name="text">
                <string>n=</string>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton">
               <property name="geometry">
                <rect>
                 <x>550</x>
                 <y>10</y>
                 <width>121</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>仪器偏差功效函数图</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>180</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>图表</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QWidget" name="paintView" native="true">
               <property name="minimumSize">
                <size>
                 <width>320</width>
                 <height>250</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QwtPlot" name="totalChart">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>180</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
